src/pages/workspace/categories/WorkspaceCategoriesPage.tsx

```diff
@@ -337,7 +339,7 @@ function WorkspaceCategoriesPage({route}: WorkspaceCategoriesPageProps) {
 
     const getHeaderText = () => (
         <View style={[styles.ph5, styles.pb5, styles.pt3, shouldUseNarrowLayout ? styles.workspaceSectionMobile : styles.workspaceSection]}>
-            {!hasSyncError && isConnectedToAccounting ? (
+            {!hasSyncError && isConnectedToAccounting && connectedIntegration ? (
```

Comment:
What's the difference between `isConnectedToAccounting` and `connectedIntegration`?